apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes.ex

```diff
@@ -280,7 +292,7 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes do
     |> String.reverse()
     |> String.graphemes()
     |> Enum.reduce_while([], fn c, acc ->
-      if String.match?(c, ~r/\s/) do
+      if String.match?(c, ~r/[\s\(\[\{]/) do
```

Comment:
This was needed if the suggested test is changed to "toPipe", because `Logger.info(to_string(...)` doesn't have any spaces, but the first `(` actually breaks the identifier `to_string`